integration_tests/packages.yml

```diff
@@ -0,0 +1,4 @@
+packages:
+  - local: ../
+  - package: fivetran/fivetran_utils
+    version: [">=0.4.0", "<0.5.0"]
```

Comment:
This is only needed in integration_tests to run `drop_schemas_automation` in the test script.